src/app/components/NumberPickers/NumberPickers.tsx

```diff
@@ -0,0 +1,14 @@
+import { getAllData } from '../../test-form/actions/form-actions';
```

Comment:
I don't think this file exists?

Reply:
Yes this was for the amplify data, it will be off